### PR TITLE
Burger menu implementing

### PR DESCRIPTION
**Pull request by VitalikVoicix**

### CODE REVIEW

File: src/components/app/app.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import { ThemeProvider } from '@material-ui/styles';
 import { CircleArrow as ScrollUpButton } from 'react-scroll-up-button';
 import { CssBaseline } from '@material-ui/core';
-
+import { getBurgerMenu } from '../../redux/burgerMenu/burgerMenu.actions';
```
Comment (ecl1pseo) [inline src/components/app/app.js:6]: Rename this to `getBurgerMenuLinks` or something like this

File: src/redux/burgerMenu/burgerMenu.actions.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {
+  GET_BURGER_MENU,
```
Comment (ecl1pseo) [inline src/redux/burgerMenu/burgerMenu.actions.js:2]: Rename it to `GET_BURGER_MENU_LINKS`

File: src/redux/burgerMenu/burgerMenu.actions.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  SET_BURGER_MENU,
```
Comment (ecl1pseo) [inline src/redux/burgerMenu/burgerMenu.actions.js:3]: Rename it to `SET_BURGER_MENU_LINKS`

File: src/redux/burgerMenu/burgerMenu.sagas.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import { setBurgerMenu, setBurgerMenuLoading } from './burgerMenu.actions';
+import { setError } from '../error/error.actions';
+import getItems from '../../utils/client';
+import { GET_BURGER_MENU } from './burgerMenu.types';
+
+export function* handleBurgerMenuLoad() {
+  yield put(setBurgerMenuLoading(true));
+  const query = `query {
```
Comment (ecl1pseo) [inline src/redux/burgerMenu/burgerMenu.sagas.js:10]: Move this to `burgerMenu.operations `folder

**Review by maksym-strus — APPROVED** [review]

**Comment by DmytroDidukh** [comment]
Please, rename all `burgerMenu` redux files to according **kebab-case** style

**Review by DmytroDidukh — APPROVED** [review]
Please, rename all `burgerMenu` files to according **kebab-case** style

File: src/redux/burgerMenu/burgerMenu.actions.js (hunk 3 above)
Comment (VitalikVoicix) [reply src/redux/burgerMenu/burgerMenu.actions.js:3]: Allready changed

Comment (VitalikVoicix) [reply src/redux/burgerMenu/burgerMenu.actions.js:3]: Allready changed

File: src/redux/burgerMenu/burgerMenu.sagas.js (hunk 4 above)
Comment (VitalikVoicix) [reply src/redux/burgerMenu/burgerMenu.sagas.js:10]: Allready move

File: src/containers/sidebar/sidebar-item/sidebar-item.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+      <Collapse in={isListOpen} timeout='auto' unmountOnExit>
+        <List className={styles.list}>
+          {models.map((model) => (
+            <ListItem
+              button
+              className={styles.nested}
+              key={model._id}
+              onClick={() => {
+                handler();
```
Comment (TorskyM) [inline src/containers/sidebar/sidebar-item/sidebar-item.js:45]: Use proper naming for this prop, it should reflect what the method does

Comment (VitalikVoicix) [reply src/containers/sidebar/sidebar-item/sidebar-item.js]: Renamed

File: src/components/app/app.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import { ThemeProvider } from '@material-ui/styles';
 import { CircleArrow as ScrollUpButton } from 'react-scroll-up-button';
 import { CssBaseline } from '@material-ui/core';
-
+import { getBurgerMenu } from '../../redux/burger-menu/burger-menu.actions';
```
Comment (ecl1pseo) [inline src/components/app/app.js:6]: rename it to `getBurgerMenuLinks`

**Review by ecl1pseo — CHANGES_REQUESTED** [review]

File: src/components/app/app.js (hunk 6 above)
Comment (VitalikVoicix) [reply src/components/app/app.js]: Renamed

File: src/components/app/app.js (hunk 1 above)
Comment (VitalikVoicix) [reply src/components/app/app.js]: Renamed

File: src/redux/burgerMenu/burgerMenu.actions.js (hunk 2 above)
Comment (VitalikVoicix) [reply src/redux/burgerMenu/burgerMenu.actions.js]: Renamed

**Review by maksym-strus — APPROVED** [review]

**Review by ecl1pseo — APPROVED** [review]

File: src/containers/sidebar/sidebar-item/sidebar-item.js [hunk 7]
```diff
@@ -22,6 +20,7 @@ const SideBarItem = ({ handlerItem, models, language, name }) => {
   const dispatch = useDispatch();
   const handleModelClick = (models) => {
     dispatch(setModelsFilter(models.name[language].value));
+    console.log(name.value);
```
[the remark this answers is not in the document]
Comment (VitalikVoicix) [reply src/containers/sidebar/sidebar-item/sidebar-item.js]: Allready removed

**Review by TorskyM — APPROVED** [review]

**Pull request merged**